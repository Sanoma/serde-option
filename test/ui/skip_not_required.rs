use serde::{Deserialize, Serialize};
use serde_option::serde_option;
use utoipa::ToSchema;

#[serde_option]
#[derive(Deserialize, Serialize, ToSchema)]
struct Foo {
    #[not_required]
    #[serde(skip)]
    x: Option<u64>,
}

fn main() {}
