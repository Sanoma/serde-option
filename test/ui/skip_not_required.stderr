error: `#[not_required]` cannot be used in combination with `#[serde(skip)]`
 --> test/ui/skip_not_required.rs:9:5
  |
9 |     #[serde(skip)]
  |     ^
